sched/worker/worker.go

```diff
@@ -0,0 +1,47 @@
+package worker
+
+import (
+	"github.com/scootdev/scoot/cloud/cluster"
+	"github.com/scootdev/scoot/sched"
+)
+
+type WorkerFactory func(node cluster.Node) WorkerController
```

Comment:
Comment here will be great for readability, and to describe what the intended behavior of functions that implement this type do.